modular_zubbers/code/modules/hydroponics/code/grown/rocks.dm

```diff
@@ -91,3 +92,421 @@
 
 /datum/reagent/consumable/nutriment/mineral
 	taste_description = "rocks and stones"
+
+
+//Rockfruits evolutions? OREFRUITS!//
+//Sand - Base tier breaks into 4 trees ('energy', Precious, Metal, Miscmats)
+/obj/item/seeds/sandfruit
+	name = "sandfruit seed pack"
+	desc = "These seeds grow to produce sandfruits."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/seeds.dmi'
+	icon_state = "seed-sandfruit"
+	species = "ore"
+	plantname = "Sandfruits"
+	product = /obj/item/food/grown/shell/sand
+	mutatelist = list(/obj/item/seeds/uraniberry,
+					/obj/item/seeds/aubergine,
+					/obj/item/seeds/ferrotuber,
+					/obj/item/seeds/adamapple)
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1,
+						/datum/reagent/silicon = 0.1)
+	growthstages = 2
+	growing_icon = 'modular_zubbers/code/modules/hydroponics/icons/growing_fruits.dmi'
+	genes = list(/datum/plant_gene/trait/chem_cooling)
+
+/obj/item/food/grown/shell/sand
+	seed = /obj/item/seeds/sandfruit
+	name = "sandfruit"
+	desc = "A mutated variant of rockfruits; rough, course and now available everywhere. Just peel it for a core."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/harvest.dmi'
+	icon_state = "orefruit"
+	trash_type = /obj/item/grown/sandfruitcore
+	foodtypes = FRUIT
+	distill_reagent = /datum/reagent/silicon
+
+/obj/item/grown/sandfruitcore
+	seed = /obj/item/seeds/sandfruit
+	name = "sandfruit core"
+	desc = "A very fragile sandfruit core, literally composed of dozens of particles of sand... don't store in pockets."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/harvest.dmi'
+	icon_state = "sandfruit"
+	custom_materials = list(/datum/material/glass =SHEET_MATERIAL_AMOUNT*0.2)
+
+//Uranium - First on 'energy' line
+
+/obj/item/seeds/uraniberry
+	name = "uraniberry seed pack"
+	desc = "These seeds grow to produce 'berries' that god thinks probably shouldn't exist."
+	icon = 'modular_zubbers/code/modules/hydroponics/icons/seeds.dmi'
+	icon_state = "seed-uraniberry"
+	species = "ore"
+	plantname = "Uraniberry"
+	product = /obj/item/food/grown/shell/uraniberry
+	mutatelist = list(/obj/item/seeds/plasmaplum)
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1,
+						/datum/reagent/uranium = 0.1)
```

Comment:
I wonder if this'll allow for EMP plants?

Reply:
It'd be like bananas and watermelons, they'd cause an instant reaction on harvest, but it'd be localised to the person dumb enough to harvest them. You can already do this with glowberries.